addons/mail/static/src/discuss/call/common/call.xml

```diff
@@ -17,41 +18,45 @@
+                <div t-if="state.overlay or !isControllerFloating" class="o-discuss-Call-overlay d-flex w-100 pb-1" t-att-class="{ 'o-isFloating position-absolute bottom-0 pb-3': isControllerFloating,'justify-content-center': !state.isCompactCall ,'justify-content-end': state.isCompactCall }">
                     <div t-on-mousemove="onMousemoveOverlay">
-                        <CallActionList thread="props.thread" compact="props.compact" fullscreen="{ isActive: state.isFullscreen, enter: () => this.enterFullScreen(), exit: () => this.exitFullScreen() }"/>
+                        <CallActionList isCompactCall="!state.isCompactCall" thread="props.thread" compact="props.compact" fullscreen="{ isActive: state.isFullscreen, enter: () => this.enterFullScreen(), exit: () => this.exitFullScreen() }"/>
                     </div>
                 </div>
                 <div t-if="hasCallNotifications" class="position-absolute d-flex flex-column-reverse start-0 bottom-0" t-att-class="{ 'ps-5 pb-5': state.isFullscreen, 'ps-2 pb-2': !state.isFullscreen }">
                     <span class="text-bg-800 shadow-lg rounded-1 m-1" t-att-class="{ 'p-4 fs-4': state.isFullscreen, 'p-2': !state.isFullscreen }" t-foreach="rtc.notifications.values()" t-as="notification" t-key="notification.id" t-esc="notification.text"/>
                 </div>
             </div>
+            <div t-att-class="{'position-absolute top-0 end-0': props.thread.activeRtcSession}" class="o-discuss-collapsible-btn justify-content-center align-items-start">
+                <button t-on-click="() => this.toggleCallSpace()" t-att-class="{ 'ms-3': !env.inChatWindow, 'ms-0': env.inChatWindow }" class="btn p-3 btn-link">
+                    <i class="fa-lg text-danger" t-att-class="{'oi oi-chevron-up': !state.isCompactCall, 'oi oi-chevron-down': state.isCompactCall}"/>
+                </button>
+            </div>
             <div t-if="state.sidebar and props.thread.activeRtcSession" class="o-discuss-Call-sidebar d-flex align-items-center h-100 flex-column">
                 <CallParticipantCard t-foreach="visibleCards" t-as="cardData" t-key="cardData.key"
                     cardData="cardData"
                     className="'o-discuss-Call-sidebarCard w-100'"
+                    isCompactCall="state.isCompactCall"
+                    toggleCallSpace="this.toggleCallSpace.bind(this)"
                     thread="props.thread"
                 />
             </div>
             <CallParticipantCard
                 t-if="props.thread.videoCount > 0 and state.insetCard"
                 cardData="state.insetCard"
                 className="'o-discuss-Call-mainCardStyle'"
+                isCompactCall="state.isCompactCall"
+                toggleCallSpace="this.toggleCallSpace.bind(this)"
```

Comment:
I would put in `env` rather than increase amount of props. Also in env you can reserve a single namespace for all features (it's discuss calls and it contains entries collapse call view)